epic-integration/src/main/kotlin/com/backend/plugins/EpicCommunication.kt

```diff
@@ -62,13 +65,33 @@ class EpicCommunication {
         return response.receive()
     }
 
+    suspend fun readPatient(patientId: String, outputFormat: String = "json"): HttpResponse {
+        val token: String = runBlocking { getEpicAccessToken() }
+        val response: HttpResponse =
+            client.get("https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/R4/Patient/" +
+                    patientId +
+                    "?_format=$outputFormat") {
+                headers {
+                    append(HttpHeaders.Authorization, "Bearer $token")
+                }
+            }
+        return response.receive()
+    }
+
+    fun parsePatientStringToObject(jsonMessage: String): Patient {
+        val jsonParser: IParser = ctx.newJsonParser()
+        jsonParser.setPrettyPrint(true)
+
+        return jsonParser.parseResource(Patient::class.java, jsonMessage)
+    }
+
     /**
      * Function to get a Condition resource.
      */
-    suspend fun getCondition(location: String?): Condition {
+    suspend fun getCondition(conditionId: String): Condition {
         val token: String = runBlocking { getEpicAccessToken() }
         val response: HttpResponse =
-            client.get("https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/R4/${location}?_format=json") {
+            client.get("https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/R4/Condition/${conditionId}?_format=json") {
```

Comment:
Tror denne endringen vil krasje med hvordan vi kaller getCondition i branch 104-impregnate nå. For der henter man det som står i headerens Location ved post, som er "Condition/xxx...xxx"

Reply:
Så forresten nå at det nok burde gjøres endringer i den andre branchen, så den bruker latestConditionId